python_modules/dagster/dagster/_core/storage/alembic/env.py

```diff
@@ -12,6 +12,6 @@
 target_metadata = [SqlEventLogStorageMetadata, SqlRunStorage, SqlScheduleStorage]
 
 if context.is_offline_mode():
-    run_migrations_offline(context, config, target_metadata)
+    run_migrations_offline(context, config, target_metadata)  # type: ignore  # (bad stubs)
```

Comment:
not grokking comment

Reply:
See my longer response to this review, `(bad stubs)` is an idiom used throughout codebase. Open to changing it.